### ISSUE TITLE
Gnome shell extension shows large icons

### ISSUE THREAD

**Issue by jcassee** [Keruspe/GPaste#307]
On my newly installed Ubuntu 20.04 system, the GPaste shell extension menu shows weird large icons:
![Schermafdruk van 2020-05-03 21-51-05](https://user-images.githubusercontent.com/20749/80924514-b41c2200-8d89-11ea-9200-9e1053c8c9d0.png)

I'm using a Gnome session. GPaste and the shell extension version 3.36.3 were installed from the system repository. My screen is scaled to 200%, but scaling back to 100% did not help.

Is this a known problem? Anything I can do to help debug?

**Comment by shimarulin** [comment]
I have same issue. Looks like `gnome-shell.css` don't contain `.system-menu-action` class in Gnome Shell 3.36.2. This is not an issue with this extension. I like Ark Dark theme and to fix it I add this code from Gnome Shell 3.34.1 to `gnome-shell.css` file of this theme: 

```css
.system-menu-action {
  -st-icon-style: symbolic;
  color: #eeeeec;
  border-radius: 32px;
  /* wish we could do 50% */
  padding: 13px;
  border: 1px solid #282828; }
  .system-menu-action:hover, .system-menu-action:focus {
    background-color: #3c3c3c;
    color: #eeeeec; }
  .system-menu-action:active {
    background-color: #1b6acb;
    color: #ffffff;
    border-color: #092444; }
  .system-menu-action > StIcon {
    icon-size: 16px; }
```

After restart Gnome Shell it's looking good:

![Снимок экрана от 2020-06-13 00-26-11](https://user-images.githubusercontent.com/5099498/84548339-0dbf2700-ad0e-11ea-8ff4-4d02ac7748a7.png)

I hope this information will help you.

**Comment by jcassee** [comment]
Thank you @shimarulin, that fixed the icons.

Not sure what the status of those CSS classes is. Are they part of the "public API" of Gnome Shell? In that case they broke backward compatibility. @Keruspe any thoughts? I'm fine with closing this issue, but maybe some other class has replaced `system-menu-action`.

**Comment by lkraav** [comment]
I have also noticed icon scaling problems when switching between scaled and unscaled displays, for example: built-in 4K (200%) and external 4K (100%) displays.

Usually, when I hover over them with the mouse, they've somehow realized they need to scale.

I'm still on Gnome 3.34, but looks like this hasn't been addressed on 3.36..

**Comment by Keruspe** [comment]
I think this is fixed by #325
